packages/jellyfish-transaction/src/script/defi/dftx_masternode.ts

```diff
@@ -0,0 +1,50 @@
+import { BufferComposer, ComposableBuffer } from '../../buffer/buffer_composer'
+
+// Disabling no-return-assign makes the code cleaner with the setter and getter */
+/* eslint-disable no-return-assign */
+
+/**
+ * CreateMasterNode DeFi Transaction
+ */
+export interface CreateMasterNode {
+  type: number // ----------------------------------| 1 byte, if 0x01 p2pkh, otherwise = p2wpkh
+  collateralPubKeyHash: string // ------------------| VarUInt{20 bytes}
```

Comment:
missing 1 operator

Reply:
Do not understand, can you elaborate?

Reply:
@ivan-zynesis here
the `operator` is needed as well
https://github.com/DeFiCh/ain/blob/e3db4110f2bcb41b4ae2d49e3fb669d11ce5cca8/src/masternodes/rpc_masternodes.cpp#L111-L113